lib/localisation/Cargo.toml

```
@@ -5,3 +5,6 @@ edition = "2021"
 
 [dependencies]
 nalgebra = "0.33.0"
```

```suggestion
nalgebra = { version = "0.33.0", default-features = false }
```
See: [https://nalgebra.org/docs/user_guide/wasm_and_embedded_targets/#compiling-without-standard-library](https://nalgebra.org/docs/user_guide/wasm_and_embedded_targets/#compiling-without-standard-library)